src/Compilers/CSharp/Test/Syntax/Diagnostics/DiagnosticTest.cs

```diff
@@ -347,6 +347,9 @@ public void WarningLevel_2()
                         case ErrorCode.WRN_InterpolatedStringHandlerArgumentAttributeIgnoredOnLambdaParameters:
                         case ErrorCode.WRN_CompileTimeCheckedOverflow:
                         case ErrorCode.WRN_MethGrpToNonDel:
+                        case ErrorCode.WRN_SwitchArmSubsumedIfNonNegativeLength:
+                        case ErrorCode.WRN_IsPatternAlwaysIfNonNegativeLength:
+                        case ErrorCode.WRN_IsPatternImpossibleIfNonNegativeLength:
```

Comment:
Shouldn't these be reported for level 7 only?

Reply:
LDM discussed a regular warning (as a documented breaking change), not a warning wave warning.